docs/recipes.rst

```diff
@@ -148,3 +148,29 @@ Also don't forget to rename the folder of your local working copy as Rally will
     mv teamtrackrepo default
 
 From now on, Rally will treat your repository as default and you need to run Rally with ``--track-repository=old-rally-default`` if you want to use the out-of-the-box Rally tracks.
+
+
+.. _recipe_testing_rally_against_ccr_clusters:
+
+Testing Rally against CCR Clusters using a remote metric store
+--------------------------------------------------------------
+
+Testing Rally features (such as the ``ccr-stats`` telemetry device) requiring Elasticsearch clusters configured for `cross-cluster replication <https://www.elastic.co/guide/en/elastic-stack-overview/current/ccr-getting-started.html>`_ can be a time consuming process.
+
+For this reason we've created an example under `recipes/ccr in Rally's repository <https://github.com/elastic/rally/tree/master/recipes/ccr>`_ simplifying it.
+
+Running the ``start.sh`` script requires Docker locally installed and performs the following actions:
+
+1. Starts a small single node Elasticsearch cluster locally, to serve as a :ref:`metrics store <configuration_options>`. It also starts Kibana attached to the Elasticsearch metric store cluster.
```

Comment:
Should we maybe state what "small" means? (i.e. 512MB heap for the metrics store and 1 GB heap for the regular nodes?)

Reply:
Fixed in https://github.com/elastic/rally/pull/676/commits/7202a0019a53e0220015b8928ef981d823d38241